src/Autofac.Integration.Wcf/IPerInstanceContextJitModuleContainer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Autofac.Core;
+using System.Collections.Generic;
+
+namespace Autofac.Integration.Wcf
+{
+    /// <summary>
+    /// A list of just in time (JIT) registrations. This allows for the
+    /// current OperationContext or WebOperationContext or any item that is
+    /// static per InstanceContext to be registered and usable throughout
+    /// the instance context.
+    /// </summary>
+    public interface IPerInstanceContextJitModuleContainer
```

Comment:
There's a general pattern in Autofac of calling these sorts of things "Accessor" - `IPerInstanceContextModuleAccessor` would be good. We know it's per-context so JIT is a little redundant in the name, and "accessor" will fit with the other things named as such.